contrib/for-tests/entrypoint-tester/ep.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+var cmdSuffices = []string{"cmd", "cmd-1", "cmd-2"}
+
+// This program prints the arguments it's passed and then
+// executes the arguments (if there are enough), waits
+// for the process to exit, and prints the output.
+//
+// If the program is invoked with a name which has a suffix
+// contained in the 'cmdSuffices' slice, the program will
+// exit after printing its arguments.
+func main() {
+	args := os.Args
+	fmt.Printf("%v\n", args)
+	num := len(args)
+
+	for _, suffix := range cmdSuffices {
+		if strings.HasSuffix(args[0], suffix) {
+			os.Exit(0)
+		}
+	}
+
+	var command *exec.Cmd
```

Comment:
out of curiosity, why do the sub-exec? Isn't verifying what args got called sufficient?